release notes/v0.27.0.md

```diff
@@ -0,0 +1,137 @@
+- The new `--execution-segment` and `--execution-segment-sequence` options allow easy partitioning of test runs across multiple k6 instances. Initially this applies to configuration options like number of VUs or iterations, but opens the door to test data partitioning, an often requested feature. See [#997](https://github.com/loadimpact/k6/issues/997) for more details.
+- Different executors can execute different functions other than the `default` exported one. This can be specified by the `exec` option in each `scenarios` config, and allows for more flexibility in organizing your tests, easier code reuse, building test suites, etc.
+- To allow for even greater script flexibility and code reuse, you can specify different environment variables and tags in each scenario, via the new `env` and `tags` executor options respectively.
+- k6 will now emit a new `dropped_iterations` metric in `shared-iterations`, `per-vu-iterations`, `constant-arrival-rate` and `ramping-arrival-rate` executors if it can't schedule an iteration on time, depending on the configured rates and the iteration duration ([#1529](https://github.com/loadimpact/k6/pull/1529)).
+
+
+## UX
+
+- CLI: There are separate descriptions and real-time thread-safe progress bars for each individual executor.
+- A new method to stop engine execution via the REST API ([#1352](https://github.com/loadimpact/k6/issues/1352)). Thanks @hynd!
+
+## Bugs fixed!
+- CLI: Stop `--http-debug` from exiting k6 on request dump error ([#1402](https://github.com/loadimpact/k6/issues/1402)). Thanks @berndhartzer!
+- CLI: JSON output is now less noisy ([#1469](https://github.com/loadimpact/k6/issues/1469)). Thanks @michiboo!
+- CLI: k6 doesn't exit when using `iterations` with `stages` ([#812](https://github.com/loadimpact/k6/issues/812)).
+- CLI: Mismatch in check counts in the end-of-test summary ([#1033](https://github.com/loadimpact/k6/issues/1033)).
+- Config: Better validation of `stages` ([#875](https://github.com/loadimpact/k6/issues/875)).
+- JS: Rare panics in goja ([#867](https://github.com/loadimpact/k6/issues/867)).
+- HTTP: Fix request timeout and wrong context used for pushing metrics ([#1260](https://github.com/loadimpact/k6/issues/1260)).
+- Execution: Fix sometimes skipping iterations with a `context cancelled` error when rapidly ramping up and down ([#1283](https://github.com/loadimpact/k6/issues/1283)).
+- WebSockets: Fix possible connection hanging when the main context is cancelled ([#1260](https://github.com/loadimpact/k6/issues/1542)).
+
```

Comment:
https://github.com/loadimpact/k6/pull/1439 this is probably something that can be mentioned under UX
https://github.com/loadimpact/k6/issues/1540 this as well

Reply:
https://github.com/loadimpact/k6/pull/1544 is a must :D maybe even link this https://community.k6.io/t/share-of-date-file-by-all-vus/407/5?u=mstoykov or ... maybe I should write a proper example with the new code ...

Reply:
See 6236f14b.

I didn't mention #1540 under UX since this is already mentioned in the breaking changes, though we didn't link to this issue. Let me know if you don't mind mentioning it twice, or if just linking to #1540 in the "Breaking changes" would be enough.

I'm not sure it would be beneficial to link to that forum post, it would just confuse people since that's no longer the optimal approach. Could you add a new example to the k6-docs PR and then link it in that forum thread (when it's merged) as a heads up?

Reply:
will try to do it today